modules/priceFloors.js

```diff
@@ -287,11 +290,12 @@ export function updateAdUnitsForAuction(adUnits, floorData, auctionId) {
       bid.floorData = {
         skipped: floorData.skipped,
         skipRate: floorData.skipRate,
+        floorMin: floorData.floorMin,
```

Comment:
@bszekely1 

Do we want to expose `floorMin` to bid adapters?

Reply:
There is not much a bid adapter can do with this information. So long as analytics adapters have access to it, we should be covered. Bid adapters would still receive the floor, just the information feeding that floor should be available to analytics.

Reply:
ok so if we leave this as is then a bid adapter will know at bidRequest time what the floorMin is.

As long as that is okay, we can and should leave it here.

Reply:
@robertrmartinez is anything else needed for this or is this good to go?